test/package/packageTest.nut.ts

```diff
@@ -128,6 +124,7 @@ describe('Integration tests for @salesforce/packaging library', () => {
       const projectFile = JSON.parse(dxPjsonData) as ProjectJson;
       expect(projectFile).to.have.property('packageDirectories').with.length(1);
       expect(projectFile.packageDirectories[0]).to.include.keys(['package', 'versionName', 'versionNumber']);
+      assert(isPackagingDirectory(projectFile.packageDirectories[0]));
       expect(projectFile.packageDirectories[0].package).to.equal(pkgName);
```

Comment:
these all presume packageDirectories[0] is the flavor that has the full packaging info.  asserting that via the typeguard makes everything downstream of that assertion type-aware